jobs/integr8ly/ocp3/test-suites/launcher-test.yaml

```diff
@@ -28,9 +28,17 @@
           name: EVALS_USERNAME
           default: 'evals11@example.com'
           description: 'Evals user name'
+      - string:
+          name: GH_USER
+          default: ''
+          description: 'GH default user name'
+      - string:
+          name: GH_PERSONAL_TOKEN
+          default: ''
```

Comment:
minor, but if the default value is an empty string you can simply not include it